### PR TITLE
Deprecate SyncPData Functions

### LINKED ISSUE #237: SyncPData functions should be deprecated
Functions like `<function name="EncodedSyncPData" functionID="EncodedSyncPDataID" messagetype="request" since="1.0">` are marked as deprecated in the documentation, but they should have the deprecated attribute as well.

### PR DESCRIPTION
**Pull request by ShobhitAd**
Fixes #237

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5423,6 +5423,7 @@ This notification tells an app to update the AddSubMenu or its 'sub' AddCommand
 ### EncodedSyncPData
 Message Type: **request**
 
+deprecated since 7.1.0
```
Comment (jacobkeeler) [inline README.md:5426]: Probably would have to be done in its own PR, but for formatting purposes, there should be a newline added after this.
```
-                            mark_down.write('\ndeprecated since ' + item.since)
+                            mark_down.write('\ndeprecated since {}\n'.format(item.since))
```

**Review by jacobkeeler — APPROVED** [review]

**Review by crokita — APPROVED** [review]

**Review by JulianKast — APPROVED** [review]

**Review by FrankElias77 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ShobhitAd) [reply README.md:5426]: @jacobkeeler agreed. Created a separate PR for the formatting fix https://github.com/smartdevicelink/rpc_spec/pull/312
